package.json

```diff
@@ -13,7 +13,14 @@
   ],
   "homepage": "https://www.rivescript.com/",
   "author": "Noah Petherbridge <root@kirsle.net> (https://www.kirsle.net/)",
-  "main": "src/rivescript.js",
+  "main": "dist/rivescript.js",
+  "browserlist": [
+    "last 2 versions"
+  ],
+  "engines": {
+    "node": ">=6.11.4 <=10",
```

Comment:
Generally a bad idea to lock between two versions like this. I've been through dependency hell dealing with external libs which have done the same.

Reply:
You seen not to have much experience with webpack don't you? Sometimes you have to lock between versions to avoid npm or yarn to conflict dependencies, then you adjust and remove the locks, this is my personal project, this is a workbench. I am doing tests, I have only pushed the modifications for you guys have a clearer vision of what I was doing. Don't pretend this is actually a pull request, and do let your ego enter the way, just try to learn from what is good, and also learn from what is bad, then you propose something better, and if everyones agree, there is, we all happy and with a good product.

Reply:
I actually use both webpack _and_ RiveScript in production! :) That's why I was interested in reviewing your PR for mistakes/issues. No hard feelings!